src/ORM/TableRegistry.php

```diff
@@ -184,7 +184,7 @@ public static function get($alias, array $options = [])
         }
 
         if (isset(static::$_config[$alias])) {
-            $options += static::$_config[$alias];
+            $options = static::$_config[$alias] + $options;
```

Comment:
@ADmad Then I guess my question becomes, is the problem demonstrated in https://gist.github.com/codeguy/1d2fdbe5aafa3d339e7c#comment-1474088 expected behavior?